nifty/pipeline/nifsUtils.py

```diff
@@ -456,6 +463,21 @@ def interactiveNIFSInput():
         config['mergeConfig']['use_pq_offsets'] = use_pq_offsets
         config['mergeConfig']['im3dtran'] = im3dtran
 
+        # Convert yes/no responses to True/False
+        def update(u):
+            for k, v in u.iteritems():
+                if isinstance(v, collections.Mapping):
+                    u[k] = update(u.get(k))
+                else:
+                    if u[k] == 'yes':
```

Comment:
Could use `u[k] = u[k] == 'yes'`

Reply:
Great suggestion, very clean. I don't this works here (if I understand what you're suggesting) because u[k] can take on more values than just 'yes' and 'no'. This function is looking to replace all occurrences of 'yes' and 'no' in the dictionary with True and False and leave all other values intact.